test/Chainweb/Test/Pact/PactInProcApi.hs

```diff
@@ -427,115 +442,98 @@ chainweb213Test bdb mpRefIO pact = do
         , "  (defun fselect () (select tbl (constantly true))))"
         , "(create-table tbl)"
         ]
-    runCut' = runCut testVersion bdb pact (offsetBlockTime second) zeroNoncer noMiner
 
-pact43UpgradeTest :: TestBlockDb -> IO (IORef MemPoolAccess) -> WebPactExecutionService -> IO ()
-pact43UpgradeTest bdb mpRefIO pact = do
+buildVector :: MonadIO m => [m a] -> m (V.Vector a)
```

Comment:
doesn't need `MonadIO` constraint I believe. Just `Monad` will do

Reply:
this will probably go away